src/components/download-overlay/download-overlay.tsx

```diff
@@ -97,7 +97,7 @@ const DownloadOverlay = withText({
         };
 
         const renderAttachments = () => {
-          return <AttachmentsList attachments={downloadMetadata!.attachments} downloadPluginManager={downloadPluginManager} />;
+          return <AttachmentsList  undisplayedAttachments={downloadConfig.undisplayedAttachments} attachments={downloadMetadata!.attachments} downloadPluginManager={downloadPluginManager} />;
```

Comment:
Probably better filter out `undisplayedAttachments` download-overlay.tsx instead of pass them to attachments-list.tsx and make filtering there

Reply:
yeah especially if we want to filter once, and renderAttachments will be called multiple times